src/util/types.ts

```diff
@@ -6,15 +6,16 @@ export type DbName = string
 export type DbNamespace = string
 export type EmptyFunction = () => void
 export type FunctionMap = {[key: string]: AnyFunction}
-export type Query = string | object
+export type Query = string
```

Comment:
See my comment above re: queries that may need to be objects.

Reply:
just took that all out. 

Users will use standard library as is, so don't think removing this has any immediate impact on what devs can do.

Reply:
👍